### PR TITLE
Update guide-ch_budget.xml

### PR DESCRIPTION
**Pull request by usamimikamen-sk**
Replate some itemizedlists with variablelists
    Replace -- with &mdash;
    Replace some - with &ndash;
    Replace ASCII - with &minus;

I've completed translating into Japanese.

### CODE REVIEW

**Comment by usamimikamen-sk** [comment]
According to the [Documentation Update Instructions](https://wiki.gnucash.org/wiki/Documentation_Update_Instructions#Text_conventions), I think it is better to use &minus;.

**Comment by jralls** [comment]
It says 
> The typographically correct symbol to use is − (U+2212, &minus;). The ASCII hyphen-minus - (U+002d) is commonly used and ***is also fine***. Whichever you use, be consistent, at least on the whole page. [emphasis added]

I don't see any other uses of - or `&minus;` on that page so consistency isn't an issue here.

**Comment by usamimikamen-sk** [comment]
&amp;minus; is already introduced some chapter like  https://github.com/Gnucash/gnucash-docs/blob/45ddec8919526557fbe7c8e02fb9c682180d84fd/guide/C/ch_invest.xml#L1666

So I think there is a plan to use it. And current documents will be modified in order to use it gradually. 

@fellen
Do you have a plan to use &amp;minus; instead of ASCII '-'?

**Comment by fellen** [comment]
> &minus; is already introduced some chapter like
> 
> https://github.com/Gnucash/gnucash-docs/blob/45ddec8919526557fbe7c8e02fb9c682180d84fd/guide/C/ch_invest.xml#L1666
> 
> So I think there is a plan to use it. And current documents will be modified in order to use it gradually.
> 
> @fellen
> Do you have a plan to use &minus; instead of ASCII '-'?

guide/C/ch_invest.xml is the only place.
I have no strong opinion. As it an almost invisible change, I would set the priority to change it to low, but if you want typographical correctness before translation I will not hinder you.

**Review by fellen — CHANGES_REQUESTED** [review]
A few minor changes would be fine.
Also line 22:
It can help you to plan …

Also sections like `Creating a Budget` could become `<task>`s.

**Review by usamimikamen-sk — COMMENTED** [review]
[dtd-translate.patch.txt](https://github.com/Gnucash/gnucash-docs/files/7016300/dtd-translate.patch.txt)

I applied this patch, then create html and PDF.
But &lt;accel&gt; tag seems to do nothing.

How &lt;accel&gt; is shown in yelp?
In some cases, we need to cusomize xsl stylesheet files.

File: guide/C/ch_budgets.xml [hunk 1]
```diff
@@ -151,7 +160,7 @@
     <sect2>
       <title>Entering Budget Values</title>
 
-      <para>Now, you must enter the budget values - the amounts that you expect the account balances to change
+      <para>Now, you must enter the budget values&ndash;the amounts that you expect the account balances to change
```
[the remark this answers is not in the document]
Comment (usamimikamen-sk) [reply guide/C/ch_budgets.xml:163]: 6e5de97a

File: guide/C/ch_budgets.xml [hunk 2]
```diff
@@ -171,16 +180,15 @@
...
       actual financial transactions compare to your plan. You need to run the Budget Report.
     </para>
 
     <para>Click on
       <menuchoice>
-        <guimenu>Reports</guimenu><guisubmenu>Budget</guisubmenu> <guimenuitem>Budget
-        Report</guimenuitem>
-      </menuchoice>
-      . For each account, the Budget Report will show the budgeted and the actual amounts in two
+        <guimenu>Reports</guimenu><guisubmenu>Budget</guisubmenu> <guimenuitem>Budget Report</guimenuitem>
```
[the remark this answers is not in the document]
Comment (usamimikamen-sk) [reply guide/C/ch_budgets.xml:189]: I'll do it after PR #207 will be accepted.

**Comment by usamimikamen-sk** [comment]
> Also sections like `Creating a Budget` could become `<task>`s.
I tried to use <task> and <procedure> instead of <sect2> in 2a86e5d.
Does it match your image?

**Comment by usamimikamen-sk** [comment]
> A few minor changes would be fine.
> Also line 22:
> It can help you to plan …

What do you mean? Is line 22 correct?
 
> Also sections like `Creating a Budget` could become `<task>`s.
Done.

**Comment by fellen** [comment]
> How <accel> is shown in yelp?

image

**Comment by fellen** [comment]
> Does it match your image?

Excellent!
But you could do more formalization by 

- converting  `<para>Before you begin…` into the taskprerequisites and
-  find parts belonging into tasksummary …

**Comment by fellen** [comment]
> > It can help you to plan …
> 
> What do you mean? Is line 22 correct?

`You can use it to help you plan …` has some redundancy—3 verbs. So i suggested
`It can help you to plan …`

**Comment by usamimikamen-sk** [comment]
> image
Thank you.

TODO:
HTML/FO `<accel>` tag underline.
Create accel-{fo,html}.xsl to apply CSS to <accel> tag.

**Pull request closed without merging**